### DISCUSSION TITLE
Rotation

### DISCUSSION THREAD

**Discussion by Ashtaavakra** [T8RIN/ImageToolbox#1414] [Ideas]
Rotation of image at custom angle is missing. Also, placing an image on another image option is not available. For example I've small image of an object and I want to place it on bigger image and adjust its location and rotate at "custom angle". I couldn't find feature to do this. Otherwise, app is very good.

**Comment by T8RIN** [comment]
Rotation is available in cropper

**Reply by Ashtaavakra** [reply]
I meant custom angle rotation. Which we do by grabbing image handle. Like rotating 45° or 15°.

**Reply by T8RIN** [reply]
Open cropper and you will see

**Reply by Ashtaavakra** [reply]
> Open cropper and you will see

No, Custom angle rotation is not there.

**Reply by T8RIN** [reply]
It is, you can rotate from -45 to 45, or rotate by 90, and perform the same

**Reply by T8RIN** [reply]
![Screenshot_20241029-211704.png](https://github.com/user-attachments/assets/ecec0c9f-3568-45f5-acb5-02f428c0a00e)

**Reply by Ashtaavakra** [reply]
Is it available in Russian version? Which version of Application? 

I installed from f-droid and version is - Version 3.0.0 (152) 


![Screenshot_2024-10-31-22-14-31-602_ru tech imageresizershrinker](https://github.com/user-attachments/assets/c820be2e-2aad-4360-9e6e-dc0305fc2675)
![Screenshot_2024-10-31-22-14-41-169_ru tech imageresizershrinker](https://github.com/user-attachments/assets/1aad315b-bbf8-4f04-acc1-41489ff91a7c)

**Reply by T8RIN** [reply]
As i remember this is part of 3.1.0

**Comment by Alshwmyhmdan1** [comment]
45
